packages/mgt-components/src/components/mgt-todo/mgt-todo.ts

```diff
@@ -251,8 +252,18 @@ export class MgtTodo extends MgtTasksBase {
      `;
   };
 
+  /**
+   * Handle a change in taskList.
+   *
+   * @protected
+   * @param {CustomEvent} e
+   * @returns {TemplateResult}
+   * @memberof MgtTodo
+   */
+
   protected handleSelectionChanged = async (e: CustomEvent) => {
-    const list = e.detail;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    const list: TodoTaskList = e.detail;
     this.currentList = list;
     await this.loadTasks(list);
```

Comment:
can we type the detail in the signature of the method and avoid the eslint suppression? 

by marking the loadTasks call as void we can also make this method synchronous and eliminate the suppression on line 114

e.g.
```suggestion
  protected handleSelectionChanged = (e: CustomEvent<TodoTaskList>) => {
    const list: TodoTaskList = e.detail;
    this.currentList = list;
    void this.loadTasks(list);
```